### DISCUSSION TITLE
New virtual user for every request using quick?

### DISCUSSION THREAD

**Discussion by kendaleiv** [artilleryio/artillery#960] [Artillery]
Is it possible to have a new connection / virtual user for every request issued by `artillery quick --rate=10 --duration=600 https://example.org/health`?

The idea is to monitor deployment for continual success responses and there may be an issue with keeping connections alive when used in this manner.

Sending the `Connection: close` request header might work? May not be able to do this with the `quick` command though.

**Comment by hassy** [comment]
hi @kendaleiv👋 Artillery opens a new connection for every VU by default (to mimic real world clients), so it should just work.

If you'd like more powerful monitoring beyond just `200 OK`, this plugin may be handy:

https://github.com/artilleryio/artillery-plugin-expect
